src/main/scala/mesosphere/marathon/api/TaskKiller.scala

```diff
@@ -55,14 +55,14 @@ class TaskKiller @Inject() (
   }
 
   private[this] def expunge(tasks: Iterable[Instance])(implicit ec: ExecutionContext): Future[Unit] = {
-    // Note: We process all tasks sequentially.
+    // Note: We process all instances sequentially.
 
-    tasks.foldLeft(Future.successful(())) { (resultSoFar, nextTask) =>
+    tasks.foldLeft(Future.successful(())) { (resultSoFar, nextInstance) =>
```

Comment:
I have no idea why, the note doesn't tell unfortunately. I'll see what I can do given time constraints, but I think this will be a follow up for later